app/assets/javascripts/discourse/app/components/discourse-banner.js

```diff
@@ -1,15 +1,18 @@
 import Component from "@ember/component";
 import discourseComputed from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
 
 export default Component.extend({
   hide: false,
 
   @discourseComputed("banner.html")
   content(bannerHtml) {
-    const $div = $("<div>");
-    $div.append(bannerHtml);
-    $div.find("[id^='heading--']").removeAttr("id");
-    return $div.html();
+    const newDiv = document.createElement("div");
+    newDiv.innerHTML = bannerHtml;
```

Comment:
(I dunno how consistent we want to be about using `domFromString()`, especially in cases like this where it won't be really any shorter 😃)

Reply:
Note that domfromstring is using a different technique which would for example not load images before being added to DOM, generally speaking I think using domfromstring should be recommended.

Reply:
Since we need to clean up headings here, we can't easily use `domFromString`, so will leave this as is. Note taken for general usage though.